### PR TITLE
Close reason, new layout, and exclusion of package-lock.json

### PR DESCRIPTION
**Pull request by mads03dk**
## Summary

* Don't include package-lock.json, as it can interfere with npm and yarn on other archs. E.g. arm
* Make default layout fit 1080p screen better (in my experience)
* Add sell reason to "Closed trades" list, so i don't have to look at telegram for the sell reason

## What's new

**New dashboard layout:**
image

**New column in closed trades:**
image

### CODE REVIEW

File: src/store/modules/layout.ts
Comment (xmatthias) [inline src/store/modules/layout.ts:41]: While on the one hand it seems more even now (so in general, i like it), there's a lot fo wasted space in the left hand column, while the chart is now ... kinda smallish (that's why i had the top layout at 3:9 instead of 4:8 in the first place).
Is there any particular reason you think the chart should be smaller than it is? (side note: The remaining fixes to this layout are great!)


The only field in "multi-pane" that benefits from this seems to be "status" - which is probably the one people look at the least - as it only provides some "quick info" data.

Comment (mads03dk) [reply src/store/modules/layout.ts:41]: I generally don't use the chart in frequi that often as it cannot be as realtime as in the exchange itself.
But the reason i have a wider mulitpane is because i seem to have a lot of folating points in my daily stats, which is pretty unreadable while being narrower:

image

I'll be looking into something to filter out folating points, and average them up for this reason.

Comment (xmatthias) [reply src/store/modules/layout.ts:41]: hm, there SHOULD be a "formatnumber" (*name might be wrong) that's applied and default-rounds it to 8 decimals.
Search for it in `formatters.ts` ... (i'm sorry, i don't remember the proper name - but it's used in the trades display too).

Comment (xmatthias) [reply src/store/modules/layout.ts:41]: @mads03dk - maybe you can merge the current master branch into this version - i've fixed the "daily" view (in da3460c) now so maybe we can then reduce this to a width of 3 again ... ?

Comment (gacopl) [reply src/store/modules/layout.ts:41]: one thing i noticed is I don't see closed trades that were closed as loss from stoploss

Comment (xmatthias) [reply src/store/modules/layout.ts:41]: @gacopl Let's not abuse a Pull request for reporting issues please, thanks.

Comment (gacopl) [reply src/store/modules/layout.ts:41]: @xmatthias yes let's add feature to already bugged part that makes more sense :) wasn't trying to abuse anything just give you or the requester a hint take it or leave it

Comment (mads03dk) [reply src/store/modules/layout.ts:41]: I've been busy lately. I will have a look at [`da3460c`](https://github.com/freqtrade/frequi/commit/da3460ca0f3089ff3a81ff53bf44f1bbd812a858) 

 @gacpol
> one thing i noticed is I don't see closed trades that were closed as loss from stoploss

I have no problem seeing trades closed by stoploss, nor trailing stoploss? Close reason is sent in the same way for all reasons since it's more or less just a DB query
image
image

Comment (gacopl) [reply src/store/modules/layout.ts:41]: ok sorry guys i found in the forrest of trades i could not see it because table sorts by trade id not closed date my stupid ux assumption

anyways thanks for great work both on frequi and this PR will not abuse in future :)

**Review by xmatthias — APPROVED** [review]
:+1: thanks a  lot!

Will fix the lint-error after merging, as you have "allow edits from maintainers" disabled :smile:

**Comment by mads03dk** [comment]
> :+1: thanks a  lot!
> 
> Will fix the lint-error after merging, as you have "allow edits from maintainers" disabled :smile: 

No problem. Yeah i forgot to enable that when I forked it. 

Next feature on my to-do list for frequi will be environment variables set in docker compose. E.g. default api URL and similar

**Comment by xmatthias** [comment]
reconsider this maybe ... the UI is now build into freqtrade (develop) - so the "seperate" compose-file will no longer be necessary - or at least won't be used by many (https://github.com/freqtrade/freqtrade/pull/4285)...

**Comment by mads03dk** [comment]
Oh. I did not realize that. Thanks for the heads up :)

**Pull request merged**
